server/src/main/java/io/crate/execution/dsl/projection/builder/InputColumns.java

```diff
@@ -224,6 +225,20 @@ protected Symbol visitSymbol(Symbol symbol, SourceSymbols sourceSymbols) {
         return inputColumn;
     }
 
+    @Override
+    public Symbol visitVoidReference(VoidReference symbol, SourceSymbols sourceSymbols) {
```

Comment:
Can this call into `visitReference` ? Or even depend on the default traversal of either `DefaultTraversalSymbolVisitor` or `SymbolVisitor` ?